### PR TITLE
Newsweek Replica Using Bootstrap

### PR DESCRIPTION
**Pull request by martin-cervantes1**
✔️ Use semantic HTML 👍 
 ✔️ Use Bootstrap CSS classes wherever it is possible 💪 
 ✔️ HTML and CSS validators 💯

### CODE REVIEW

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @mcervantes71  👋

You've done really well on this project in terms of the frontend structure and layout of the webpage.🚀

However, there are just a few issues that need to be fixed before proceeding:

- [x] Required - Kindly enable stickler on your repo.

- [x] Required - In addition, consider removing the `!important` tag that exists on your CSS codebase. We generally want to avoid CSS bad practices refer to this ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright).

- [x] Required - Also kindly remove the inline styles that exist in your HTML code, refer also to the article mentioned above👆

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: css/style.css [hunk 1]
```diff
@@ -134,9 +137,861 @@ table {
...
+}
+
+.show_md {
+  display: block;
+}
+
+/* more rules */
+
+.bg_gray0 {
+  background-color: #f4f5f7 !important;
```
Comment (mikenath223) [inline css/style.css]: Please remove this `!important` tag as well as other `!important` tags that exist in your CSS code.

File: index.html [hunk 2]
```diff
@@ -5,14 +5,1278 @@
...
+
+                            <div class="py-2">
+                                <span class="megazine_section">DOWNTIME</span><a class="megazine_category" href="#">CULTURE</a>
+                            </div>
+
+                            <h2 class="py-2"><a class="megazine_link_heading2" href="#">10 Books That Will Transport You All Around the World</a></h2>
+                        </article>
+
+                        <article class="col-md-3 pb-3">
+                            <div class="megazine_article_img" style="background-image: url('../img/megazine8.jpg')"></div>
```
Comment (mikenath223) [inline index.html]: Kindly remove the inline styling that exists here and other inline styles also.

File: css/style.css (hunk 1 above)
Comment (martin-cervantes1) [reply css/style.css]: Stickler is enable.
What could be the problem ??

**Comment by martin-cervantes1** [comment]
☑️ Enable stickler in the repository. 👌 
☑️ Refactor the CSS file to remove the !important tags that existed 💪 
☑️ Refactor inline styles that exist in your HTML ✍️

**Comment by uryelah** [comment]
Hello **Martin Cervantes Palomo**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all great job with your attention to detail, it's really nice how you added the navbar search detail on desktop screens and the text over the images🌟**

There are a couple of issues you need to fix before going to the next project, remember this project needs to be extra closer to the original since Bootstrap is already taking care of the responsiveness and most of the positioning, so keep calm and code on 💪 

---------     

## Required Changes 🏗 👷‍♀ 

- [x] Before the 'In the Magazine' section there should be no extra spaces from the above sections. This is easily fixable by setting a max-height together with overflow scroll or even a position sticky if you want a challenge. 
Compare the space above the section in your clone and in the original project:

Screenshot_2020-04-29 Newsweek - News, Analysis, Politics, Business, Technology(1)
Screenshot_2020-04-29 Newsweek(4)

----

- [x] The 'In the Magazine' first article image is taller than the following ones but its bottom should still be aligned with the other articles, compare your version with the original one:
Screenshot_2020-04-29 Newsweek - News, Analysis, Politics, Business, Technology
Screenshot_2020-04-29 Newsweek(3)

---

- [x] Mind the font-weight of the section titles, 'World', 'US', etc, they should be bolder. Tip: you can use Bootstrap to change text weight too.

- [x] Make sure your navbar if fixed at the top of the page, it should be visible when scrolling down.

- [x] Remember to keep using semantic HTML. Use `section` tags for sections and `article` tags for articles. You can [read this if you need a refresher on semantic HTML.](https://www.w3schools.com/html/html5_semantic_elements.asp)

----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by martin-cervantes1** [comment]
✔️  Position sticky to reduce extra space before magazine section 🎨 
✔️ Magazine' first article image is taller than the following ones 📐 
✔️ Change font-weight using bootstrap class (title, categories, title section, etc) ⚡ 
✔️ Make navbar fixed ⭐

**Comment by trekab** [comment]
# Project Approved! :ok_hand:
Well done, @mcervantes71! __Let's get this merged.__ :tada:

_Goodluck with your next projects and happy coding!_

**Pull request merged**